logger.log

```diff
@@ -0,0 +1,475 @@
+2020-10-16 14:15:38 INFO  Version:46 - HHH000412: Hibernate Core {5.4.5.Final}
```

Comment:
this file should not be here
in should be located in `logs` folder that is added to gitignore

let's remove the file from the PR